### DISCUSSION TITLE
Downgrading back to stock A6

### DISCUSSION THREAD

**Discussion by tbilic14** [Mr-MIBonk/M.I.B._More-Incredible-Bash#189] [Q&A]
Hi all,

My car had an MHI2_ER_AU57x_P3634 and I updated with MHI2_ER_AU57x_K3663_1_AIO based on instructions by mr-fix I found on youtube. Then I discovered this awesome tool... It seems that it cannot patch the current FW I have, dual navigation is not working and wlan client activation is not working. I was thinking maybe going back to P3634 and then doing MiB thingies from scratch.

Here I was just wondering on advices what I should be careful about to minimize possibility to mess something up... I read something about sound system might not work and needs some additional coding...

Any comments much appreciated before I go for it.

**Comment by harman-f** [comment]
No need to downgrade
AIO provied the same patch as this solution.

Navignore is sadly not supported by your FW version :(
